src/app/page.tsx

```diff
@@ -11,13 +11,26 @@ import { Categories } from './constants';
 import useFullScreen from '@/custom-hooks/use-full-screen';
 import Header from '@/components/header';
 import { Icons } from '@/components/ui/icons';
+import { Label } from '@/components/ui/label';
 
 export default function Page() {
   const [activeElement, setActiveElement] = useState<Item | null>(null);
   const [open, setOpen] = useState(false);
   const [textSearch, setTextSearch] = useState('');
   const [activeCategory, setActiveCategory] = useState<Categories | null>(null);
   const { toggleFullScreen, isFullScreen } = useFullScreen();
+  const [isMobile, setIsMobile] = useState(window.innerWidth <= 768);
```

Comment:
we have a useMobile custom hook that we should import instead. Although I'm not sure we even need to detect if the user is on their mobile device or not since we're just changing text.

We should just use media queries via tailwind instead